docs/extend/index.md

```diff
@@ -213,7 +213,7 @@ $ git clone https://github.com/vieux/docker-volume-sshfs
 $ cd docker-volume-sshfs
 $ docker build -t rootfs .
 $ id=$(docker create rootfs true) # id was cd851ce43a403 when the image was created
-$ sudo mkdir -p /var/lib/docker/plugins/$id/rootfs
+$ sudo mkdir -p /var/lib/docker/plugins/$id/rootfs/run/docker/plugins
```

Comment:
the L208 comment confused me because it wasn't clear if it was on the host or in the container.  Also the plugin helper library doesn't always make this directory (or at least it didn't for me in the plugin) which is why I wasn't sure.  Maybe that comment can be clarified and more clear that it's the responsibility of the plugin to create that directory?

Or we can just make it in these instructions -- I think we should decide based on how `docker plugin build` will handle it.